<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTunePI</class>
 <widget class="QMainWindow" name="FormTunePI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tune PI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>350</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ki</string>
    </property>
   </widget>
   <widget class="QELabel" name="lblKP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QELabel" name="lblKi">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QENumericEdit" name="txtKP">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="writeOnChange">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QENumericEdit" name="txtKI">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="writeOnChange">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
